### DISCUSSION TITLE
Crowdsource Translations with Hosted Weblate

### DISCUSSION THREAD

**Discussion by ejektaflex** [ejektaflex/Bountiful#219] [Mod Ideas]
I'm currently working on a setup that will allow us to crowdsource translations via CrowdIn. This way, we can more easily support multiple languages! :)

**Comment by comradekingu** [comment]
Is it possible to use Hosted Weblate, where the rest of the Minecraft mods are?
Crowdin has egregious terms, and is missing a lot of essential functionality like navigating strings and seeing more languages at once.

**Reply by ejektaflex** [reply]
That's a good idea - I'll look into it, thank you! I wasn't aware of it's existence 🙂
